Procedural Dungeon Generator

```diff
@@ -1,29 +1,43 @@
-    This will act as a step-by-step checklist to help people build unique maps.
+    This will act as a step-by-step list of guidelines to help people build fun, functioning 
+maps. It may be a good idea to have a few pages of notes detailing any special rules 
+for your map, as well as the inventories, gear, and personalities of your characters.
 
 
 ==============================
 The Base Map
 ==============================
     
         Firstly, determine the planet or region this map will take place in, as 
-well as any area-specific rules that may apply. Some examples include the map 
-being on a water world with most or all of the environment underwater, in space 
-with no air or gravity, in a snowstorm with very limited visibility but also the 
-ability to track people based on their footprints, on a volcanic planet where 
-staying outside too long without a mask can injure or blind the players, etc. 
-If no specific rules exist for the map you want to make, it will be up to you to 
-create them. In general, map-specific rules should not be a major hinderance to 
-the players, but should make the experience more interesting and challenging. If 
-traversing the map is harder than fighting the enemies, then it won't be a fun game
-for anybody.
+well as any area-specific rules that may apply. 
```

Comment:
Starting with a planet type is a specific implementation of the more abstract strategy of starting with a theme or idea. A lot of good maps are made by starting with an idea, such as "what if we did a map that was on a living, moving tree? Wouldn't that be cool?" 
Instead, I recommend that your first step in creating a dungeon be setting a goal. How will you know when your dungeon is finished? Dungeoneering is an inherently iterative and recursive process. If you don't set a termination point at the start, then you'll be working on your dungeon forever, and nobody will ever get to see your hard work. Suggested goals are things like "I want this dungeon to be fun for every type of player in Bartle's Taxonomy." Another good list of goals are in the dungeon festival rubric.